### DISCUSSION TITLE
Better export extension : need help

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#16457] [Feature Requests]
I am building an extension in vuejs : 
From a collection, I read all fields, then export it to csv.
For Files field, I download all files.

Then,
A zip is created with the csv and the files (pdf jpg etc) in  a "doc" folder.

So, here is my question...

Is there a way I can intercept the current collection export style ? Or add a different export collection.

It will be the best option;..

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
